src/components/modal/modal.e2e.ts

```diff
@@ -509,7 +509,29 @@ describe("calcite-modal accessibility checks", () => {
       const scrim = document.querySelector("calcite-modal").shadowRoot.querySelector(".scrim");
       return window.getComputedStyle(scrim).getPropertyValue("--calcite-scrim-background");
     });
-    expect(scrimStyles).toEqual("rgba(0, 0, 0, 0.75)");
+    expect(scrimStyles).toEqual(" rgba(0, 0, 0, 0.85)");
```

Comment:
For some reason there is an extra space being returned with the new fallback css var setup, but the value is correct.

![Screen Shot 2022-12-16 at 8 24 33 PM](https://user-images.githubusercontent.com/4733155/208225610-a6d81ab3-3c5a-4222-99ff-74a9c0762a32.png)

Reply:
Are we able to fix the fallback css var setup instead of doing this?

Reply:
I tried but I think it gets added during css processing, unless I’m missing something obvious. We could ‘trim()’ the expected value in the test, although it’s not ideal either way.

Reply:
We could maybe just do `expect(scrimStyles.trim()).toEqual`

Reply:
Yeah just going with `trim()`, the value is correctly set - I think since the first value in the fallback sequence is not provided it's processed as an empty space.